README.md

```diff
@@ -1,62 +1,82 @@
-# Getting started with CPU Stats on Mbed OS
+![](./resources/official_armmbed_example_badge.png)
+# CPU Statistics Mbed OS Example
 
-This guide reviews the steps required to get CPU statistics on Mbed OS platform.
+The example project is part of the [Arm Mbed OS Official Examples](https://os.mbed.com/code/) and is the [getting started example for Mbed OS](https://os.mbed.com/docs/mbed-os/development/quick-start/index.html).
 
-Please install [mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+You can build the project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-## Import the example application
+1. [Install Mbed CLI](https://os.mbed.com/docs/mbed-os/latest/quick-start/offline-with-mbed-cli.html).
 
-From the command-line, import the example:
+1. Clone this repository on your system, and change the current directory to where the project was cloned:
 
-```
-mbed import mbed-os-example-cpu-stats
-cd mbed-os-example-cpu-stats
-```
+    ```bash
+    $ git clone git@github.com:armmbed/mbed-os-example-cpu-stats && cd mbed-os-example-cpu-stats
+    ```
 
-### Now compile
+    Alternatively, you can download the example project with Arm Mbed CLI using the `import` subcommand:
 
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
+    ```bash
+    $ mbed import mbed-os-example-cpu-stats && cd mbed-os-example-cpu-stats
+    ```
 
-```
-mbed compile -m K64F -t ARM
-```
 
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
+## Application functionality
+
+The `main()` function creates an event queue and adds a function to periodically print the time spent while up, in idle mode, in sleep mode, in deep sleep mode and usage information about the CPU. It also starts a thread that blinks an LED at varying intervals to keep the CPU busy.
```

Comment:
Add that the first 4 times are since the system started  and that the last 2 are CPU usage information since the last call.
I would separate the two sets in two bullets for readability.